src/Monticello/ChangeRecord.extension.st

```diff
@@ -10,7 +10,7 @@ ChangeRecord >> asMCMethodDefinition [
 		className: class
 		classIsMeta: meta
 		selector: self methodSelector
-		category: category
+		category: protocol
```

Comment:
the keyword is still `category:` -_-

Reply:
Yes but MCMethodDefinition is next on the list ahah

I'm trying to do small PRs to help with review and not take too much risk of breaking everything.